src/careamics/models/lvae/noise_models.py

```diff
@@ -2,6 +2,7 @@
 
 from typing import TYPE_CHECKING, Optional
 
+import matplotlib.pyplot as plt
```

Comment:
So far we have always avoided depending on matplotlib directly. This makes the CI fail on this PR. 

In addition, I am not sure it should be the responsibility of the NM to plot itself, I'd rather have this type of function in a utility class rather than buried in essential code. Details, the first comment is more important.

Reply:
Removed matplotlib as discussed